src/common/src/config.rs

```diff
@@ -721,6 +721,9 @@ pub struct StorageConfig {
     #[serde(default = "default::storage::mem_table_spill_threshold")]
     pub mem_table_spill_threshold: usize,
 
+    #[serde(default = "default::storage::memory_limit_for_behind_barrier_ratio")]
+    pub memory_limit_for_behind_barrier_ratio: u64,
```

Comment:
The naming is confusing. From the code, it might mean `memory_limiter_fast_quota_ratio`?

Reply:
This configuration has been removed